test/constants.test.ts

```diff
@@ -0,0 +1,26 @@
+import { v4 } from 'uuid';
```

Comment:
You can move this file in `/constants`, be careful with the build version (it should keep the same structure).